test/src/parse/test_replace.cpp

```diff
@@ -0,0 +1,165 @@
+
+#include <gtest/gtest.h>
+
+#include <parse/parselo.h>
+
+#include "globalincs/safe_strings.h"
+
+#include "util/FSTestFixture.h"
+
+class ReplaceTest : public test::FSTestFixture {
+ public:
+	ReplaceTest() : test::FSTestFixture(INIT_MOD_TABLE | INIT_CFILE) {
+		pushModDir("parselo");
+	}
+
+ protected:
+	void SetUp() override {
+		test::FSTestFixture::SetUp();
+	}
+	void TearDown() override {
+		stop_parse();
+
+		test::FSTestFixture::TearDown();
+	}
+};
+
+TEST_F(ReplaceTest, replace_one_cstr)
+{
+	constexpr int len = 10;
```

Comment:
Maybe `LEN` or `MAX_LEN`, since it's a `constexpr` integral value?

Reply:
K